autopush/main.py

```diff
@@ -323,7 +323,7 @@ class RustConnectionApplication(AutopushMultiService):
     logger_name = "AutopushRust"
     push_server = None
 
-    def setup(self, rotate_tables=True, num_threads=10):
+    def setup(self, rotate_tables=True, num_threads=10, json_logging=True):
```

Comment:
you should put the argparse'd human_logs setting into AutopushConfig. Then AutopushServer can set its cfg.json_logging from it. Then you won't need the various json_logging arguments/passthrus